tests/js/client/server_parameters/foxx-enable-true.js

```diff
@@ -31,7 +31,20 @@ if (getOptions === true) {
   };
 }
 const jsunity = require('jsunity');
+const internal = require('internal');
+const fs = require('fs');
+const path = require('path');
+var utils = require('@arangodb/foxx/manager-utils');
```

Comment:
`var` should not be used in any new code.
```suggestion
const utils = require('@arangodb/foxx/manager-utils');
```